lib/api/streamer.js

```diff
@@ -8,11 +8,12 @@ var _ = require('underscore');
 var util = require('util');
 var EventEmitter = require('events').EventEmitter;
 
-var QUERY_INTERVAL = 100; // 100ms
+var QUERY_INTERVAL = 2000; // 2s
 
 var Streamer = function() {
   this.queries = [];
   this.status = 'idle';
+  this.throttledQuery = _.throttle(this.query, QUERY_INTERVAL);
```

Comment:
The throttledQuery was added because currently `query` is being executed every time a new report is being saved.